lib/src/boilerplate_suggestors/props_meta_migrator.dart

```diff
@@ -38,6 +38,11 @@ class PropsMetaMigrator extends GeneralizingAstVisitor
 
   PropsMetaMigrator(this.converter);
 
+  // Utility to keep track of when a `const` keyword is removed from the left side of a `TypedLiteral`
+  // so that we can prevent "overlapping" patches when a literal contains more than one `PrefixedIdentifier`
+  // - which causes the codemod script to crash.
+  static final Map<TypedLiteral, bool> _literalsWithConstRemoved = {};
```

Comment:
This doesn't need to be `static`, and could have its scope limited; it's only really applicable within a single traversal of a given `VariableDeclaration`.

Also, having this be an `Expando` would probably be preferable so that this Map doesn't retain any memory unnecessarily across codemods:
```suggestion
  final Expando<bool> _constRemovedFlag = Expando();
  bool _hasLiteralHadConstRemoved(TypedLiteral literal) = _constRemovedFlag[literal] ?? false;
```